simulations/vip-130/testnet/simulations.ts

```diff
@@ -0,0 +1,6 @@
+import { forking, testVip } from "../../../src/vip-framework";
+import { vip130Testnet } from "../../../vips/vip-130/testnet";
+
+forking(30855643, () => {
+  testVip("VIP-130-testnet Swap routers accept ownership", vip130Testnet());
```

Comment:
I would check the emitted events, as we did [here](https://github.com/VenusProtocol/vips/pull/26/files#diff-45f08afe7b6cdfa73f61a4eb7af923662e56464c55253dd7bc8d5b9d21d62a2dR60), for example

Reply:
Added, 7e4cd21462d18171f65759acdda628511d6949d6